pkg/operator/defaultstorageclass/controller.go

```diff
@@ -177,6 +191,9 @@ func newStorageClassForCluster(infrastructure *configv1.Infrastructure) (*storag
 	case configv1.VSpherePlatformType:
 		return resourceread.ReadStorageClassV1OrDie(generated.MustAsset("storageclasses/vsphere.yaml")), nil
 	default:
+		if csi.SupportedCSIPlatform(infrastructure.Status.PlatformStatus.Type) {
```

Comment:
Maybe this would be enough that's needed in this PR:
`case configv1.OVirtPlatformType: return nil`.

I don't like having some platforms listed in `supportedCSIDrivers` array and another list in this `switch`.

Reply:
I removed the supportedCSIDrivers slice and just added the check for the OvirtPlatformType. If we remove the default StorageClass entries for other platforms we can simply update the call here.